src/test/java/com/eta/houzezbackend/controller/AgentControllerTests.java

```diff
@@ -6,14 +6,16 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.http.MediaType;
 
 
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-class AgentControllerTests extends ApplicationIntTest{
+@AutoConfigureMockMvc(addFilters = false)
```

Comment:
Move this annotation to` ApplicationIntTest`

Reply:
done